lib/mix/tasks/deploy.prod.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Mix.Tasks.Deploy.Prod do
+  @moduledoc """
+  Open the Github Releases page that triggers a production deploy
+  """
+  use Mix.Task
+
+  @shortdoc "Open the Github Releases page that allows us to trigger a production deploy"
+  @spec run([binary]) :: any
+  def run(_) do
+    System.cmd("git", ["fetch"])
+
+    latest_tag =
+      System.cmd("git", ["describe", "origin/main", "--tags", "--abbrev=0"])
+      |> case do
+        {tag, 0} -> tag
+      end
+      |> String.trim()
+
+    %{"previous_date" => previous_date, "previous_count" => previous_count} =
+      Regex.named_captures(~r/(?<previous_date>.*)-(?<previous_count>\d)/, latest_tag)
+
+    today = Date.to_string(Date.utc_today())
+
+    count =
+      1 +
+        case today do
```

Comment:
There's a part of me that thinks that this deserves a test, but I don't think we typically give deploy scripts automated tests.

I'll think about whether it's worth it.